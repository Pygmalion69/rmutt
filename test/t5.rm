// repetition
t: "a"{2} "b"{2,3} "c"? "d"* "e"+;