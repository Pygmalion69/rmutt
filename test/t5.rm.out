aabbbcddddde
aabbcddddde
aabbbddddde
aabbddddde
aabbbcdddde
aabbcdddde
aabbbdddde
aabbdddde
aabbbcddde
aabbcddde
aabbbddde
aabbddde
aabbbcdde
aabbcdde
aabbbdde
aabbdde
aabbbcde
aabbcde
aabbbde
aabbde
aabbbce
aabbce
aabbbe
aabbe
